test/java/seedu/addressbook/data/person/NameTest.java

```diff
@@ -8,7 +8,6 @@
 	
 	@Test
 	public void nothing() {
-		fail("Hi");
```

Comment:
Yupp, similar comment for you do refer to my comment from the previous commit, but i'm glad you're finding it fun & interesting !